Src/WitsmlExplorer.Frontend/contexts/modificationStateReducer.tsx

```diff
@@ -123,12 +123,14 @@ const addWell = (state: NavigationState, { payload }: AddWellAction) => {
 };
 
 const updateWell = (state: NavigationState, { payload }: UpdateWellAction) => {
-  const { well } = payload;
+  const well = payload.well;
+  const overrideWellbores = payload.overrideWellbores;
```

Comment:
Change these two to `const { well, overrideWellbores } = payload;`

Reply:
Fixed.